Getting Started with Splitter (#6266)
DevExpress/devextreme-documentation: concepts/05 UI Components/Splitter/00 Getting Started with Splitter/15 Fill Panes with Content.md

Splitter panes can include different content types, from simple HTML markup to components. You can declare HTML markup inside the item tag or use the following properties to populate panes with content:

- [text](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#text)   

> Specifies text displayed for the pane.

Reviewer: ```suggestion
Specifies the text displayed in the pane.
```

- [itemTemplate](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/#itemTemplate)    
Specifies a custom template for all panes.

- [template](/Documentation/ApiReference/UI_Components/dxSplitter/Configuration/items/#template)    
Specifies a custom template for an individual pane.

> This tutorial uses **templates** to define components inside panes. The first pane includes a Form component. All changes made in Form are recorded in the second pane. The third pane has a button. When you click it, the output in the second pane is erased, and the Form is reset. The last pane demonstrates the usage of the **text** property. 

Reviewer: ```suggestion
This tutorial uses **templates** to define components inside panes. The first pane includes a Form component. All changes made in Form are recorded in the second pane. The third pane contains a button. Users can click it to erase the output in the second pane and reset the Form. The last pane demonstrates the use of the **text** property. 
```
